lib/knapsack_pro/runners/queue/base_runner.rb

```diff
@@ -77,6 +77,8 @@ def log_threads
           puts 'Use the following backtrace(s) to find the line of code that got stuck if the CI node hung and terminated your tests.'
           puts 'How to read the backtrace: https://knapsackpro.com/perma/ruby/backtrace-debugging'
 
+          post_log_threads(threads)
```

Comment:
plea

"Post" does not work because this happens in the middle of `log_threads`. What about `log_current_tests`?

Also, we are using both log and print, maybe we should converge into one?

Reply:
There's an inconsistency: `print_current_batch_rspec_command` uses `puts`, `log_current_tests` uses `puts`.

So they both use `puts` but why is one called "print" and the other "log"?